docs/config.c

```diff
@@ -241,7 +241,7 @@
 ** of the value as shown above if included.
 */
 
-{ "attach_format", DT_STRING, "%u%D%I %t%4n %T%.40d%> [%.7m/%.10M, %.6e%?C?, %C?, %s] " },
+{ "attach_format", DT_STRING, "%u%D%I %t%4n %T%d %> [%.7m/%.10M, %.6e%<C?, %C>, %s] " },
```

Comment:
is the removal of `.40d` intended?

Reply:
Yes, this is just syncing it with the code value.
We dropped the restriction back in 1541755c2 2023-02-26.
The following `%> ` expando means that the right-hand block will automatically truncate the `%d` on low-space.